Tests/Marketplace/search_and_uninstall_pack.py

```diff
@@ -11,6 +11,8 @@
 from Tests.Marketplace.search_and_install_packs import install_packs
 from time import sleep
 
+UNREMOVABLE_PACKS = ['Base', 'CoreAlertFields', 'Core']
```

Comment:
Are there mandatory dependencies for these packs?

Reply:
yes, for **Core**:
<img width="803" alt="image" src="https://github.com/demisto/content/assets/46249224/7eec4a22-be3f-408b-b2aa-ff4e12e657f0">
and for **CoreAlertFields**:
<img width="803" alt="image" src="https://github.com/demisto/content/assets/46249224/cae300f0-0874-41f9-b308-c461dc2aa8b6">